URL-API.md

```diff
@@ -4,77 +4,289 @@ MazeMap URL API
 This document describes the basic use of the MazeMap URL API.
 
 
-0. Specifying the URL API version
+Specifying a specific URL API version
 --------------------
-The URL API supports versioning using the _**v**_ parameter, however it is not necessary to specify, as the API assumes the latest version by default. There are legacy functionality from version 0.5 that still works. More on that further down.
 
 ```
 http://use.mazemap.com/?v=1
 ```
 
+* `v` defines the version, in this case `1`.
 
-1. Specifying a campus
+The URL API supports versioning using the `v` parameter, however it is not necessary to specify as the API assumes the latest version by default. In the following examples we omit this parameter. There is legacy functionality from version 0.5 that still works. More on that further down.
+
+
+Specifying a campus
 --------------------
-To start mazemap with a specific campus in view, use the _**campusid**_ parameter. For other parameters, you must also specify the campus to provide proper context.
```

Comment:
Consistency: if using the text format `campusid` somewhere else, do not use _**campusid**_